README.md

```diff
@@ -1,3 +1,21 @@
 # Signadot Examples
 
 This repository contains examples for Signadot users.
+
+# Setup Instructions for Running Examples
+
+1. Ensure you have [Docker](https://www.docker.com/), [minikube](https://minikube.sigs.k8s.io/docs/), [helm](https://helm.sh/), and [kubectl](kubectl apply -f https://raw.githubusercontent.com/signadot/hotrod/main/k8s/all-in-one/demo.yaml) installed.
+2. Run [`minikube start`](https://minikube.sigs.k8s.io/docs/start/) to create a Kubernetes environment locally.
+3. Create our all in one demo application in your local Kubernetes cluster by running `kubectl apply -f https://raw.githubusercontent.com/signadot/hotrod/main/k8s/all-in-one/demo.yaml`.
+4. Create a cluster in the [Signadot Dashboard](https://app.signadot.com/) and make sure to copy the token generated for it for the next step.
+5. Run the following commands:
+    1. `helm repo add signadot https://charts.signadot.com`
+    2. `helm install signadot-workspaces signadot/workspaces`
+    3. `kubectl -n signadot create secret generic cluster-agent --from-literal=token=...` where the `...` is replaced with the token you copied when creating the cluster in the Signadot Dashboard.
+6. Create a new workspace in the Signadot Dashboard in the newly-created cluster. Select "hotrod" as the namespace, and choose "route" in the "Select Deployments to Fork" section.
```

Comment:
The step 6 and beyond following this are not needed - after this, I'd expect them to just run the integration tests. I was thinking of these 5 steps above to be the pre-requisites section in https://github.com/signadot/examples/blob/main/java-sdk-example/README.md and https://github.com/signadot/examples/blob/main/node-sdk-example/README.md

And then the next step for a user is to run the integration tests.

Reply:
Gotcha. Your comment about using existing images based on hotrod to spin up new workspaces threw me off here.